content/docs/introduction/billing-overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Billing overview
+enableTableOfContents: true
+subtitle: The basics about how billing works in Neon
+---
+
+Just as we separate storage from compute in our architecture, we separate storage from compute when calculating your bill. The largest part of your bill typically comes from compute costs &#8212; the size of your compute plus your level of activity. Storage costs make up the next largest part of your bill. More minimally, the amount of data that you write, either to storage within Neon or data egress outside of Neon, also adds to your cost, though typically a small percentage per month.
+
+## Compute costs
+With key features like [autoscaling](/docs/guides/autoscaling-guide), [auto-suspend](/docs/guides/auto-suspend-guide), minimum and max [compute sizes](/docs/manage/endpoints#compute-size-and-autoscaling-configuration) enabled, you can get a sense of how your compute costs might accrue in the following graph.
+
+![Compute metrics graph](/docs/introduction/compute-metrics2.png)
+
+You can see how your the compute time that you get billed for scales between your minimum and maximum CPU settings: compute time never rises above your max level ,and it never drops below your minimum setting. With autosuspend, you are not charged at all during inactive periods. For projects with inconsitent demand, this can save significant compute costs.
```

Comment:
```suggestion
You can see how the compute time that you get billed for scales between your minimum and maximum CPU settings: compute time never rises above your max level ,and it never drops below your minimum setting. With autosuspend, you are not charged at all during inactive periods. For projects with inconsistent demand, this can save significant compute costs.
```